packages/playwright-core/src/server/registry/index.ts

```diff
@@ -743,7 +761,8 @@ export class Registry {
   }
 
   private _downloadURLs(descriptor: BrowsersJSONDescriptor): string[] {
-    const downloadPathTemplate: string|undefined = (DOWNLOAD_PATHS as any)[descriptor.name][hostPlatform];
+    const paths = (DOWNLOAD_PATHS as any)[descriptor.name];
+    const downloadPathTemplate: string|undefined = paths[hostPlatform] || paths['<unknown>'];
```

Comment:
I think this has a side-effect when downloading browsers, probably for unknown linux versions?

Reply:
I don't think so?